sycl/test/matrix/matrix-bfloat16-test-coord-basicA.cpp

```diff
@@ -0,0 +1,203 @@
+// RUN: %clangxx -fsycl -O2 -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 %s -o %t.out
```

Comment:
I think these tests should only run on Intel devices supporting AMX and XMX. @dkhaldi ?

Reply:
I am just compiling the code for the test case.